FinniversKit/Sources/Components/Label/Label.swift

```diff
@@ -74,28 +74,3 @@ extension Label {
         UIMenuController.shared.setMenuVisible(true, animated: true)
     }
 }
-
-// MARK: - Support HTML
-
-public extension Label {
-    func setText(fromHTMLString htmlString: String, style: [String: String] = [:]) {
-        var styledText = htmlString
```

Comment:
Just a heads up: removing this method makes this a breaking change, and not a minor bump.